charts/ccd-data-store-api/values.preview.template.yaml

```diff
@@ -16,12 +16,12 @@ java:
     DATA_STORE_DB_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
     DATA_STORE_DB_OPTIONS: "?stringtype=unspecified"
     DATA_STORE_DB_MAX_POOL_SIZE: 10
-    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575.service.core-compute-preview.internal/
+    DEFINITION_STORE_HOST: http://ccd-definition-store-api-PR-891.service.core-compute-preview.internal/
     USER_PROFILE_HOST: http://ccd-user-profile-api-pr-335.service.core-compute-preview.internal/
     ENABLE_DB_MIGRATE: true
 
-    ELASTIC_SEARCH_ENABLED: false # enable whenever ES required on a particular PR
-    ELASTIC_SEARCH_NODES_DISCOVERY_ENABLED: false
+    ELASTIC_SEARCH_ENABLED: true # enable whenever ES required on a particular PR
```

Comment:
this means we are enabling ES func tests on every PR now?

Reply:
This is mainly for testing this PR only, Tests on PR's wont work 'out of the box' so I would suggest either 
this gets set to false before merge or we follow up with another pr to set ELASTIC_SEARCH_ENABLED: false after master build is green

Reply:
ok got it cheers